src/components/dialog/dialog.js

```diff
@@ -1,59 +1,68 @@
 /**
  * @function UIDialog
  * @version 0.0.2
- * @desc display a Dialog
  */
 
 export default function UIDialog(userOptions) {
 
-    var DOM,
+    let DOM,
         state = {};
-        
+
     const defaults = {
         dialog:         '.js-dialog',
         openBtn:        '.js-dialog-btn',
         closeBtn:       '.js-dialog-close-btn',
-        backdrop:       '.js-dialog-backdrop',
         isModal:        false,
         showBackdrop:   true
     };
 
     // Combine defaults with passed in options
     const settings = Object.assign(defaults, userOptions);
-    
+
+    // Create the backdrop
+    const backdrop = document.createElement('div');
+
+    backdrop.classList.add('dialog-backdrop');
+
```

Comment:
I guess we're going to be creating a few elements with this project, could be worth creating a helper for it at some point